purchase_deposit/models/purchase.py

```diff
@@ -27,8 +27,8 @@ class PurchaseOrderLine(models.Model):
         " order. They are not copied when duplicating a purchase order.",
     )
 
-    def _prepare_account_move_line(self, move):
-        res = super(PurchaseOrderLine, self)._prepare_account_move_line(move)
+    def _prepare_account_move_line(self):
```

Comment:
@kayue I think you can not remove param `move` from the function like this. It was there in the core odoo.
https://github.com/odoo/odoo/blob/14.0/addons/purchase/models/purchase.py#L1092

Reply:
Can you fix this and also rebase your branch, it was quite old. :)